### DISCUSSION TITLE
%E4%BD%BF%E7%94%A8hugo-github%E6%90%AD%E5%BB%BA%E5%8D%9A%E5%AE%A2/

### DISCUSSION THREAD

**Discussion by giscus** [williamlfang/williamlfang.github.io#4] [Announcements]
# %E4%BD%BF%E7%94%A8hugo-github%E6%90%AD%E5%BB%BA%E5%8D%9A%E5%AE%A2/

使用 Hugo + Github 搭建个人博客系统，并托管在 Github Page 服务。

https://williamlfang.github.io/%E4%BD%BF%E7%94%A8hugo-github%E6%90%AD%E5%BB%BA%E5%8D%9A%E5%AE%A2/

**Comment by williamlfang** [comment]
利用周末的时间，把原来博客的主题更新成 Hugo FixIt，并整理了以前的部分博客内容。

- FixIt 针对代码块有做过优化，在高亮、长代码块的自动折叠等方面还是不错的
- 现在的博客系统接入了 ~~Gitalk~~ Giscus 评论系统，与 Github 相关联
- 去掉与博客写作无关的模块，更加专注于写作
- 对于 Hugo 有了进一步的了解，后面可以考虑自己设计前端
- 添加博文的月历功能，方便查看对应日期的博客
